llvm/lib/Frontend/OpenMP/OMPIRBuilder.cpp

```diff
@@ -5655,8 +5657,8 @@ GlobalVariable *
 OpenMPIRBuilder::createOffloadMapnames(SmallVectorImpl<llvm::Constant *> &Names,
                                        std::string VarName) {
   llvm::Constant *MapNamesArrayInit = llvm::ConstantArray::get(
-      llvm::ArrayType::get(
-          llvm::PointerType::getUnqual(M.getContext()), Names.size()),
+      llvm::ArrayType::get(llvm::PointerType::getUnqual(M.getContext()),
+                           Names.size()),
```

Comment:
It looks like you clang-formated the entire file. Either only format your changes, or pre-commit a NFC that formats the existing file. This way it clutters the changes with unrelated ones.

Reply:
Oh, sorry. I assumed the file would be formatted. I will update this soon.